services/thermal_mgr/thermal_mgr.c

```diff
@@ -42,19 +42,38 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 }
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
-  /* Send an event to the thermal manager queue */
-
+  if (xQueueSend(thermalMgrQueueHandle, event, (TickType_t) 0) != pdPASS) {
```

Comment:
- [ ] check that the queue handle is not null before use, returning invalid state
- [ ] check that the event is not null before use